bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -165,6 +165,13 @@ namespace JsRTApiTest
         JsValueRef value2 = JS_INVALID_REFERENCE;
         REQUIRE(JsPointerToString(_u("value1"), wcslen(_u("value1")), &value2) == JsNoError);
 
+		// Test JsPointerToString and JsCreateString on NULL inputs
+		JsValueRef nullStr;
+		REQUIRE(JsPointerToString(NULL, 0, &nullStr) == JsNoError);
+
+		JsValueRef nullFname;
+		REQUIRE(JsCreateString(NULL, 0, &nullFname) == JsNoError);
+
```

Comment:
Please could you use spaces instead of tabs for the indent (here and other files) - our code base uses spaces consistently.